app/cdash/include/CDash/Database.php

```diff
@@ -62,80 +62,46 @@ public function getPdo()
     }
 
     /**
-     * @param \PDOStatement $stmt
-     * @param null $input_parameters
-     * @return bool
-     *
      * @deprecated 04/22/2023  Use Laravel query builder or Eloquent instead
      */
-    public function execute(\PDOStatement $stmt, $input_parameters = null)
+    public function execute(PDOStatement $stmt, ?array $input_parameters = null): bool
```

Comment:
Nitpick: This and ln81 were the same argument before now they have different signatures.  Will the `= null` and `=[]` really be equivalent?

Reply:
Got it.  Thanks for the clarification @williamjallen